scripts/settlement/register_rollapp_to_hub.sh

```diff
@@ -15,19 +15,13 @@ if [ -z "$HUB_CHAIN_ID" ]; then
   HUB_CHAIN_ID="dymension_100-1"
 fi
 
-# this file is generated using the scripts/settlement/generate_denom_metadata.sh
-DENOM_METADATA_PATH="${ROLLAPP_SETTLEMENT_INIT_DIR_PATH}/denommetadata.json"
-# this file is generated using the scripts/settlement/add_genesis_accounts.sh
-GENESIS_ACCOUNTS_PATH="${ROLLAPP_SETTLEMENT_INIT_DIR_PATH}/genesis_accounts.json"
-
 set -x
 #Register rollapp
 dymd tx rollapp create-rollapp "$ROLLAPP_CHAIN_ID" "$MAX_SEQUENCERS" '{"Addresses":[]}' \
-  "$DENOM_METADATA_PATH" \
-  --genesis-accounts-path "$GENESIS_ACCOUNTS_PATH" \
   --from "$DEPLOYER" \
   --keyring-backend test \
   --broadcast-mode block \
-  --fees 1dym
+  --fees 1dym \
```

Comment:
shouldn't this also have `--transfers-enabled true` ?

Reply:
no
but now it's different anyway after
https://github.com/dymensionxyz/dymension/pull/961
https://github.com/dymensionxyz/dymension-rdk/pull/458